conformance/binary_json_conformance_suite.cc

```diff
@@ -2289,17 +2289,17 @@ void BinaryAndJsonConformanceSuiteImpl<
   // Duplicated field names are not allowed.
   ExpectParseFailureForJson("FieldNameDuplicate", RECOMMENDED,
                             R"({
-        "optionalNestedMessage": {a: 1},
+        "optionalNestedMessage": {\"a\": 1},
```

Comment:
nit: this is a raw string, you don't need to escape quotes

Reply:
Not a nit at all, as it turns out.